realm/realm-library/src/objectServer/java/io/realm/internal/network/LogoutResponse.java

```diff
@@ -79,16 +81,6 @@ private LogoutResponse(String serverResponse) {
      * @return {@code true} if valid.
      */
     public boolean isValid() {
-//        return (error == null);
-        return true;
-    }
-
-    /**
-     * Returns the error.
-     *
-     * @return the error.
-     */
-    public ObjectServerError getError() {
-        return error;
+        return (error == null) || (error.getErrorCode() == ErrorCode.EXPIRED_REFRESH_TOKEN);
```

Comment:
`valid` in response means success of request.

https://github.com/realm/realm-java/blob/my/implement_global_logout/realm/realm-library/src/objectServer/java/io/realm/internal/network/ExponentialBackoffTask.java#L33

If the response is *NOT* valid, retry will happen and that is not what we want.

https://github.com/realm/realm-java/blob/my/implement_global_logout/realm/realm-library/src/objectServer/java/io/realm/internal/network/ExponentialBackoffTask.java#L68